plugins/servicenow/icon_servicenow/connection/connection.py

```diff
@@ -18,10 +18,7 @@ def connect(self, params):
         api_route = "api/now/"
         incident_table = "incident"
 
-        self.base_url = params.get(Input.URL, "")
-
-        if not self.base_url.endswith("/"):
-            self.base_url = f"{self.base_url}/"
+        self.base_url = f"https://{params.get(Input.INSTANCE, '')}.service-now.com/"
```

Comment:
I know the URL now needs to be hardcoded in order for it to work with the cloud orch

By changing this to instance does this solve that issue?